### PR TITLE
Update anticheat-compatibility.md

### PR DESCRIPTION
**Pull request by Shad0wolf0**
Updated most anti-cheats to the latest update as of 11th February 2023. Verus update found on their discord.
Reflex was removed from Spigot, nowhere else to be found. More info here: https://www.spigotmc.org/threads/reflex-anticheat.572425

### CODE REVIEW

File: _docs/geyser/anticheat-compatibility.md [hunk 1]
```diff
@@ -10,27 +10,26 @@ Do note that this is a community-compiled list and is incomplete! If you wish to
...
-- [Flappy Anticheat](https://www.spigotmc.org/resources/92180/){:target="_blank"} - Last Updated on 14th September 2021
-- [GrimAC](https://github.com/MWHunter/Grim){:target="_blank"} - Last Updated on 24th March 2022
-- [Matrix](https://matrix.rip/){:target="_blank"} (Paid) - Last updated on 2nd September 2021
-- [Vulcan](https://www.spigotmc.org/resources/83626/){:target="_blank"} (Paid) - Last Updated on 19th August 2021
-- [Verus](https://verus.ac){:target="_blank"} (Paid) - Last Updated on 13th July 2022
+- [Flappy Anticheat](https://www.spigotmc.org/resources/92180/){:target="_blank"} - Last Updated on 15th November 2022
+- [GrimAC](https://github.com/MWHunter/Grim){:target="_blank"} - Last Updated on 7th Februrary 2023
+- [Matrix](https://matrix.rip/){:target="_blank"} (Paid) - Last updated on 5th October 2022
+- [Vulcan](https://www.spigotmc.org/resources/83626/){:target="_blank"} (Paid) - Last Updated on 12th December 2022
+- [Verus](https://verus.ac){:target="_blank"} (Paid) - Last Updated on 10th August 2022
```
Comment (Chew) [inline _docs/geyser/anticheat-compatibility.md:26]: Can you put these in alphabetical order? :)

File: _docs/geyser/anticheat-compatibility.md [hunk 2]
```diff
@@ -10,27 +10,26 @@ Do note that this is a community-compiled list and is incomplete! If you wish to
...
-- [Kauri](https://www.spigotmc.org/resources/53721/){:target="_blank"} (Paid) - Last Updated on 21st March 2022
-- [Reflex](https://www.spigotmc.org/resources/21122/){:target="_blank"} (Paid) - Last Updated on 19th August 2021
-- [Wraith](https://www.spigotmc.org/resources/66887/){:target="_blank"} (Paid) - Last Updated on 19th August 2021
+- [AntiCheatReloaded](https://www.spigotmc.org/resources/23799/){:target="_blank"} - Last Updated on 1st Janurary 2022
+- [Negativity V1](https://www.spigotmc.org/resources/48399/){:target="_blank"} - Last Updated on 4th January 2023
+- [NoCheatPlus](https://ci.codemc.io/job/Updated-NoCheatPlus/job/Updated-NoCheatPlus/){:target="_blank"} ([Compat NCP](https://github.com/Updated-NoCheatPlus/CompatNoCheatPlus/){:target="_blank"} is an addon that enables Partial Compatibility) - Last Updated on 19th January 2023
+- [GodsEye](https://www.spigotmc.org/resources/69595/){:target="_blank"} ([GodsEyeGeyserMC](https://github.com/TheDejavu/GodsEyeGeyserMC/releases){:target="_blank"} is an addon that enables Partial Compatibility) (Paid) - Last Updated on 12th January 2023
```
Comment (Chew) [inline _docs/geyser/anticheat-compatibility.md:33]: Same with this

**Review by Shad0wolf0 — COMMENTED** [review]
got it (:

**Comment by Kas-tle** [comment]
Apologies for the lack of action on this. Is this still up to date?

**Comment by Chew** [comment]
Needs a rebase regardless

**Comment by Konicai** [comment]
This became stale unfortunately - will be reopened if updated

**Pull request closed without merging**